content/recipes/introduction/fairification-process.md

```diff
@@ -162,22 +261,25 @@ While this recipe does not deal in detail with how to implement the FAIRificatio
 ## Conclusion
 ***NOTE: this is a direct c/p from Box 1 in the process paper. Do not publish as is!***
 
-1. Focus on incrementally achievable targets. Projects approach the FAIR principles in different orders and risk overdoing. Instead, we focus on achieving elements of FAIRness that matter most to the needs of the project to reach a balanced “FAIR enough” status.
-1. Tailor the generic FAIRification process to individual needs. Projects have different needs, even when the underlying data, capability and resource requirements appear to be quite similar. Customising the relevant template elements allows to form a coherent workplan. 
-1. Assemble a multi-disciplinary team. Projects are often multi-partner, international and distributed in nature, with datasets of different provenance and source. A successful FAIRification process starts with bringing together diverse teams that include the data owners as well professionals who can tackle the legal, as well as curational and technical infrastructure aspects. 
+1. **Focus on incrementally achievable targets**. Projects approach the FAIR principles in different orders and risk overdoing.
+Instead, we focus on achieving elements of FAIRness that matter most to the needs of the project to reach a balanced **“FAIR enough”** status.
+2. **Tailor the generic FAIRification process to individual needs**. Projects have different needs, even when the underlying data, 
+capability and resource requirements appear to be quite similar. Customising the relevant template elements allows to form a coherent workplan. 
+3. **Assemble a multi-disciplinary team**. Projects are often multi-partner, international and distributed in nature, 
+with datasets of different provenance and source. A successful FAIRification process starts with bringing together diverse
+teams that include the data owners as well professionals who can tackle the legal, as well as curatorial and technical 
+infrastructure aspects. 
 
 ### What should I read next?
-> * [Tips and Tricks](./tips-tricks.md)
-> * [how-to-create-recipe-with-hackmd](./how-to-create-recipe-with-hackmd.md)
-> * using a reference tag affored by MyST markdown and sphinx with JupyterBook 0.7+. <!-- {ref}`fcb-intro-fair-principles` -->
-  [my-prefered-label](fcb-help-recipe-template)
+TODO
```

Comment:
I wonder if we should add here the recipes which Dani mentions in her paper alongside the table with FAIRification template..?

Reply:
Not a bad shout but it's a very long list now. I think there are only 24 recipes NOT mentioned in the table :)